src/spec/settings_spec.js

```diff
@@ -178,4 +178,26 @@ describe('Settings Page', () => {
     //Assert
     expect(settings.countDomainListItems()).toBeLessThan(countDomainListItems);
   });
+  it('should set as disabled domain', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.addMockModule('descartableModule2', () => angular
+      .module('descartableModule2', ['ngMockE2E'])
+      .run($httpBackend => {
+        $httpBackend.whenGET(/\/accounts\/[\w|-]*\/domains/).respond(200, {
+          "domains": [{name: "relay.com"}, {name: "fromdoppler.com" }, {name: "makingsense.com", disabled: true }, {name: "makingsense12.com" }],
+          "defaultDomain": "relay.com"
+        });
+        $httpBackend.whenPUT(/\/accounts\/[\w|-]*\/domains/, '{"disabled":true}').respond(200, {});
```

Comment:
I think that the URL should include the domain name. What do you think about to, as a rule, add `$` prefix and `^` sufix to regular expressions to ensure to not forgot something?